language-server/modules/langserver-core/src/test/java/org/ballerinalang/langserver/completion/ModulePartNodeContextTest.java

```diff
@@ -34,6 +38,12 @@ public void init() throws Exception {
         preLoadAndInit();
     }
 
+    @Test(dataProvider = "completion-data-provider")
+    @Override
+    public void test(String config, String configPath) throws WorkspaceDocumentException, IOException {
+        super.test(config, configPath);
+    }
+
```

Comment:
Let's remove this unwanted changes

Reply:
@malinthar If we remove this the test case won't execute during the build. Am I correct?

Reply:
We already have a data provider with the same name in the class. So we don't need this ryt?